client/clip_client/client.py

```diff
@@ -145,43 +152,46 @@ def _gather_result(self, response, results: 'DocumentArray'):
         )
 
     @staticmethod
-    def _unboxed_result(results: 'DocumentArray'):
+    def _unboxed_result(original_input, results: 'DocumentArray'):
```

Comment:
could we control the return type by an argument?

```suggestion
    def _unboxed_result(results: 'DocumentArray', return_np: bool = False):
```

Reply:
We can, but this will:
1. introduce duplicated codes
2. make `_unboxed_result` a oneliner and useless

see this -> https://github.com/jina-ai/clip-as-service/pull/815/commits/711d9efc41450a53df5c8dbceacbc9386b85121d

Reply:
If `_unboxed_result` is useless, we can remove it. I cannot see the strong reason to pass `origin_input`.

Reply:
👌